app/controllers/hosts_controller.rb

```diff
@@ -151,7 +151,7 @@ def hostgroup_or_environment_selected
 
   def current_parameters
     Taxonomy.as_taxonomy @organization, @location do
-      render :partial => "common_parameters/inherited_parameters", :locals => {:inherited_parameters => refresh_host.host_inherited_params(true)}
+      render :partial => "common_parameters/inherited_parameters", :locals => {:inherited_parameters => refresh_host.inherited_params(true)}
```

Comment:
[In hostgroup.rb](https://github.com/theforeman/foreman/pull/2649/files#diff-92b23d0f5006d635f922fa85badf077fR83) you're returning `:inherited_parameters => group.parent_params(true), :parameters => hostgroup.group_parameters }`. 

I was wondering whether this line in `hosts_controller` is not missing also a `:parameters => refresh_host.host_parameters` to include Host specific parameters, the same way Hostgroup does.